editor.planx.uk/src/@planx/components/Checklist/Public.tsx

```diff
@@ -4,7 +4,7 @@ import type { Checklist, Group } from "@planx/components/Checklist/model";
 import ImageButton from "@planx/components/shared/Buttons/ImageButton";
 import Card from "@planx/components/shared/Preview/Card";
 import QuestionHeader from "@planx/components/shared/Preview/QuestionHeader";
-import { useFormik } from "formik";
+import { getIn, useFormik } from "formik";
```

Comment:
This was required to get the correct type for the error.

Docs: https://formik.org/docs/api/fieldarray#fieldarray-validation-gotchas